doc/modules/ROOT/pages/generate-jwt.adoc

```diff
@@ -15,6 +15,18 @@ encrypting the nested JWT token.
 
 SmallRye JWT provides an API for securing the JWT claims using all of these options.
 
+== Configuration
```

Comment:
Hi @bobbyphilip Can you please make it the last section in this doc, as it would be good to start with the overview of the features and then list these 2 properties (there will be more to come)